cmd/gateway-gcs.go

```diff
@@ -381,7 +382,7 @@ func newGCSGatewayLayer(projectID string) (GatewayLayer, error) {
 	}
 
 	// Initialize a GCS client.
-	client, err := storage.NewClient(ctx)
+	client, err := storage.NewClient(ctx, option.WithUserAgent(fmt.Sprintf("Minio/%s (GPN:Minio;)", Version)))
```

Comment:
Is `GPN` google standard?

Reply:
Only link i could find is this https://github.com/GoogleCloudPlatform/google-cloud-node/issues/2659#issuecomment-334867549 - GPN (Google Partner Network) is perhaps the terminology here.

Reply:
yes @balamurugana this change was requested by them

Reply:
perhaps its better to write a comment regarding this? @krishnasrinivas